app/google-services.json

```diff
@@ -12,15 +12,33 @@
           "package_name": "com.github.se.eventradar"
         }
       },
-      "oauth_client": [],
+      "oauth_client": [
```

Comment:
It looks like these changes are still in this PR, can you move this to your other PR?

Reply:
Done, Files Changed now doesn't show this :)